packages/typespec-client-generator-core/package.json

```diff
@@ -57,13 +57,17 @@
     "pluralize": "^8.0.0"
   },
   "peerDependencies": {
+    "@azure-tools/typespec-autorest": "workspace:~",
+    "@azure-tools/typespec-azure-core": "workspace:~",
     "@typespec/compiler": "workspace:~",
     "@typespec/http": "workspace:~",
     "@typespec/rest": "workspace:~",
     "@typespec/versioning": "workspace:~"
   },
   "devDependencies": {
     "@azure-tools/typespec-azure-core": "workspace:~",
+    "@azure-tools/typespec-azure-resource-manager": "workspace:~",
```

Comment:
we can't test with arm or autorest bc of circular dependencies. cc @timotheeguerin, going to remove for this release

Reply:
how should we do test with arm? we have such flag in tcgc.

Reply:
Today we can’t because of that cicurlar dependencies issue. If this is a priority we need to speed up removing autorest from arm library